torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -34,14 +34,17 @@ def get_dimensions(image: Union[PIL.Image.Image, torch.Tensor, features.Image])
     return list(get_chw(image))
 
 
-def get_image_num_channels(image: Union[PIL.Image.Image, torch.Tensor, features.Image]) -> int:
+def get_channel_size(image: Union[PIL.Image.Image, torch.Tensor, features.Image]) -> int:
```

Comment:
I agree to remove "image" from the name, but why not

```suggestion
def get_num_channels(image: Union[PIL.Image.Image, torch.Tensor, features.Image]) -> int:
```

Reply:
Sounds good to me.